app/upgrades/v8/upgrades.go

```diff
@@ -16,8 +18,6 @@ func CreateUpgradeHandler(
 	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		ctx.Logger().Info("start to run module migrations...")
```

Comment:
why do we want to remove this? during a previous upgrade validators complained about the lack of feedback in the terminal when performing an upgrade. it took a really long time and there was no info about what was happening. @yaruwangway added this so that users would know that the upgrade was starting this step in order to track progress.

Reply:
ah i see it was just moved lower : )